From 783091a2c39c5c53dd06cd1d95c156d016d306b4 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Wed, 24 Oct 2012 11:32:34 -0500
Subject: ENGR00231266-1: fsl_devices add csi_tx_addr

* Add csi_tx_addr, so it can override slave addr
  for adv7280 tvin decoder device

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
---
 include/linux/fsl_devices.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index ab22c4a..78a50ac 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -320,6 +320,8 @@ struct fsl_mxc_tvin_platform_data {
 	void (*reset)(void);
 	void (*io_init)(void);
 	bool cvbs;
+	/* adv7280 mipi-csi i2c slave addr */
+	u8 csi_tx_addr;
 };
 
 struct mpc8xx_pcmcia_ops {
-- 
1.7.9.5

